RocketDataTests/CollectionDataProviderTests/SharedCollectionDataProviderTests.swift

```diff
@@ -790,6 +791,7 @@ class SharedCollectionDataProviderTests: SharedCollectionTests {
             finishCacheLoad = {
                 completion(initialModels, nil)
             }
```

Comment:
I think it might be simpler/clearer if you just saved the completion block itself instead of wrapping it in another closure.

```
var cacheLoadCompletion: ((/* whatever this type is */, NSError) -> Void)? // instead of finishCacheLoad
cacheDelegate.collectionForKeyCalled = { cacheKey, context, completion in
   // ...
   cacheLoadCompletion = completion
   // ...
}
```